syntax = "proto3";

package gtools.tank.query;

enum Field {
  TANK_FIELD_UNSPECIFIED = 0;
  TANK_FIELD_TYPE = 1;
  TANK_FIELD_OBJECT_TYPE = 2;
  TANK_FIELD_JUMP_COUNT = 3;
  TANK_FIELD_ANIMATION_TYPE = 4;
  TANK_FIELD_NET_ID = 5;
  TANK_FIELD_TARGET_NET_ID = 6;
  TANK_FIELD_FLAGS = 7;
  TANK_FIELD_FLOAT_VAR = 8;
  TANK_FIELD_VALUE = 9;
  TANK_FIELD_VECTOR_X = 10;
  TANK_FIELD_VECTOR_Y = 11;
  TANK_FIELD_VECTOR_X2 = 12;
  TANK_FIELD_VECTOR_Y2 = 13;
  TANK_FIELD_PARTICLE_ROTATION = 14;
  TANK_FIELD_INT_X = 15;
  TANK_FIELD_INT_Y = 16;
  TANK_FIELD_EXTENDED_LEN = 17;
}
